client/app/index.jsx

```diff
@@ -34,9 +34,9 @@ class App extends React.Component {
       params: { q: trend }
     })
     .then(response => {
-      let timeline = response.data;
+      let timeline = response.data.timeline;
       this.setState({
-        trend: trend,
+        trend: response.data.trend,
```

Comment:
omg ive wanted to capitalize this for the chart title. not sure if better here or at the chart component